### DISCUSSION TITLE
so-soc MISSING

### DISCUSSION THREAD

**Discussion by ducna96** [Security-Onion-Solutions/securityonion#7597] [Unsupported Versions]
I run so-status and  i see log so-soc MISSING
env: ubuntu 18.04 install with 
git clone https://github.com/Security-Onion-Solutions/securityonion
cd securityonion
sudo bash so-setup-network
![image](https://user-images.githubusercontent.com/35019158/158756713-d0212b75-637d-4e57-9a34-a210cfe6e2ee.png)

what wrong install?
Thanks

**Comment by dougburks** [comment]
Was this a new installation?

Were there any errors in `sosetup.log`?

Have you checked the logs in `/opt/so/log/soc/`?

**Reply by Silvrak** [reply]
I am seeing this issue as well. 

Not a new install, has been in production over 6 months as a standalone server.

Logs in /opt/so/log/soc/  pointed me to: error="unexpected end of JSON input" file=jobs/soserv/1044.json
                                                                  "                                                                                        "1045.json
                                                                  "                                                                                        "1046.json

the .json files are completely devoid of any code...no content at all.

Thanks

**Reply by Silvrak** [reply]
Sorry for lack of info: 
1) Same setup as OP

**Reply by dougburks** [reply]
@Silvrak From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.

**Reply by Silvrak** [reply]
Will do. Sorry for hijacking this thread.

**Comment by TotieBash** [comment]
Try so-docker-pull or so-docker-update, I cant remember the actual syntax.  I had similar issue I was missing so-redis.  Why the docker image disappear is a mystery I havent figure out yet.

**Comment by densnoigaskogen** [comment]
hi @ducna96 , did you manage to resolve this so-soc missing issue? I do not see any confirmed solutions for this issue.
